### PR TITLE
StringComparison

### PR DESCRIPTION
**Pull request by aparnamittal**
## Related Issue
(#1105)

## Proposed Changes
Added a file comparing two strings .
Most string comparison algorithms compare the strings character by character. C++ provides us some built in operators like == and != to compare strings along with strcmp and compare functions.

image

## Additional Info
- Any additional information or context

## Checklist

- [x] ✅ My code follows the code style of this project.
- [x] 📝 My change requires a change to the README.
- [x] 🎀 I have updated the README accordingly.
- [x] 👀 I have read the [**CONTRIBUTING**](https://github.com/Lakhankumawat/LearnCPP/blob/main/Contributing.md) document.
- [x] ✨ I have added tests to cover my changes.
- [x] 🚩 All new and existing tests passed.
- [x] 🌟 ed the repo

# Algorithm:
1. Declare two strings.
2. take the input of two strings.
3. comparing the two strings we can use any of the three ways mentioned below:

i) strcmp() inbuilt function
ii) compare() inbuilt function, 
iii) C++ relational operators ( ‘==’, ‘!=’)

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Output Screenshots
1.   when strings are equal
image

2.  when strings are not equal
image

### CODE REVIEW

**Comment by aparnamittal** [comment]
@Lakhankumawat sir you told me  to change the file name , but i am not able to make that change on the same PR so i have created new PR and made the changes.

https://github.com/Lakhankumawat/LearnCPP/pull/1112#discussion_r851043110

File: S-Strings/StringComparison.cpp [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    // Taking the input of two strings
+    cout << " Enter the String 1: " << endl;
+    cin >> string1;
+    cout << " Enter the String 2: " << endl;
+    cin >> string2;
+
+    //comparing the two strings and returning the value
+    if ( string1 == string2)
+    {
+        cout <<endl << " string are equal." << endl;
```
Comment (Mrjoy832) [inline S-Strings/StringComparison.cpp:18]: here will be **strings are** not string

Comment (aparnamittal) [reply S-Strings/StringComparison.cpp]: sir made the changes @Mrjoy832

Comment (Mrjoy832) [reply S-Strings/StringComparison.cpp]: > sir made the changes @Mrjoy832

already referred to admin..work on another issue.

**Review by Lakhankumawat — APPROVED** [review]
LGTM 🎉✨

**Comment by Lakhankumawat** [comment]
@all-contributors please add @aparnamittal code ,

**Pull request merged**